bundles/org.openhab.binding.wled/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wled"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="wled">
+		<label>WLED String</label>
+		<description>A WLED string of LEDs</description>
+		<category>ColorLight</category>
+		<channels>
+			<channel id="masterControls" typeId="masterControls"/>
+			<channel id="primaryColor" typeId="primaryColor"/>
+			<channel id="primaryWhite" typeId="primaryWhite"/>
+			<channel id="secondaryColor" typeId="secondaryColor"/>
+			<channel id="secondaryWhite" typeId="secondaryWhite"/>
+			<channel id="presets" typeId="presets"/>
+			<channel id="presetDuration" typeId="presetDuration"/>
+			<channel id="transformTime" typeId="transformTime"/>
+			<channel id="presetCycle" typeId="presetCycle"/>
+			<channel id="palettes" typeId="palettes"/>
+			<channel id="fx" typeId="fx"/>
+			<channel id="speed" typeId="speed"/>
+			<channel id="intensity" typeId="intensity"/>
+			<channel id="sleep" typeId="sleep"/>
+		</channels>
+		<config-description>
+			<parameter name="address" type="text" required="true">
+				<context>network-address</context>
```

Comment:
`network-address` should only allow IP addresses.

Reply:
DONE: removed as discussed above.